java/google/registry/rde/imports/RdeHostLinkAction.java

```diff
@@ -117,10 +117,18 @@ public void map(JaxbFragment<XjcRdeHostElement> fragment) {
         ofy().transact(new VoidWork() {
           @Override
           public void vrun() {
+            // link host to superordinate domain and set time of last superordinate change to
+            // the time of the import
             HostResource host =
                 ofy().load().now(Key.create(HostResource.class, xjcHost.getRoid()));
             ofy().save()
-                .entity(host.asBuilder().setSuperordinateDomain(superordinateDomainKey).build());
+                .entity(host.asBuilder().setSuperordinateDomain(superordinateDomainKey)
+                    .setLastSuperordinateChange(ofy().getTransactionTime())
+                    .build());
+            // link domain to subordinate host
+            DomainResource domain = ofy().load().now(superordinateDomainKey);
```

Comment:
I don't understand why I need to get the key off the host. I could get the domain/key inside the transaction, which would avoid reloading the domain.